utmap-client/src/components/pages/CalendarPage.js

```diff
@@ -11,19 +11,34 @@ import {
 } from '@devexpress/dx-react-scheduler-material-ui';
 
 //Scheduler is the calendar, today, and taskbar components
-function CalendarPage(props) {
-    return (
-        <Paper>
-            <Scheduler data={props.events}>
-            <ViewState />
-            <MonthView />
-            <Toolbar />
-            <DateNavigator />
-            <TodayButton />
-            <Appointments />
-            </Scheduler>
-        </Paper>
-    );
+class CalendarPage extends Component {
```

Comment:
You switched from function components to class component here -- we'll be using functional components for this project.